containers/themes/ThemesSection.js

```diff
@@ -3,45 +3,49 @@ import { c } from 'ttag';
 import {
     useMailSettings,
     useEventManager,
-    useApiWithoutResult,
+    useApi,
+    useLoading,
     SubTitle,
     Alert,
     ThemeCards,
     useModals,
     useNotifications
 } from 'react-components';
 import { updateTheme } from 'proton-shared/lib/api/mailSettings';
-import { getThemeIdentifier } from 'react-components/helpers/themes';
-import { THEMES } from 'proton-shared/lib/constants';
+import { getThemeIdentifier, stripThemeIdentifier } from 'proton-shared/lib/themes/helpers';
+import { ALL_THEMES } from 'proton-shared/lib/themes/themes.js';
 
-import { availableThemes } from './availableThemes.js';
 import CustomThemeModal from './CustomThemeModal.js';
 
-const {
-    CUSTOM: { identifier: customId }
-} = THEMES;
+const { DARK, CUSTOM } = ALL_THEMES;
```

Comment:
Since we plan to add a "dark" theme later, can we rename `DARK` to `NORMAL`?

Reply:
Changed to default